server/utils/routes.js

```diff
@@ -34,6 +37,9 @@ export const v1Routes = {
   dashboards: {
     show: `${v1Root}/dashboards/:study/:subject`,
   },
+  siteMetadata: {
+    save: `${v1Root}/import/data/metadata`,
```

Comment:
I believe these should be index routes to follow our routing conventions.

Reply:
[c1cf0d0](https://github.com/AMP-SCZ/dpdash/pull/382/commits/c1cf0d04b9d9dd43b6aceef72cc4f1f45af67a9c)